specs/pluggable_alert_delivery.adoc

```diff
@@ -35,16 +37,37 @@ via mediums such as:
 * Whenever a new handler is added, the first thing the handler is supposed to
   do is add itself as a means to list of supported notification mechanisms in
   etcd.
-* For the api to get list of the supported notification mechanisms, it can
-  look into this list.
 * The alerts raised are persisted in etcd before sending out notifications in
   '/alerts' directory of etcd by node-agent.
-* The alerts marked as severity critical are deemed important and hence their
+* Each alert will carry an additional attribute that carries its significance
+  which will be added by the respective handler. The supported significance
+  levels can be 'HIGH', 'MEDIUM' and 'LOW'.
```

Comment:
This is something we should be handling as clearing event for an existing previous critical event associated with it. Can you think of something on this line??